--- /usr/lib/x86_64-linux-gnu/maliit/keyboard2/qml/Keyboard.qml.orig
+++ /usr/lib/x86_64-linux-gnu/maliit/keyboard2/qml/Keyboard.qml
@@ -58,10 +58,11 @@
         objectName: "MaliitKeyboard" // Allow us to specify a specific keyboard within autopilot.

         anchors.bottom: parent.bottom
-        anchors.horizontalCenter: parent.horizontalCenter
+	anchors.left: undefined

-        width: parent.width
+        width: parent.width * 0.7
         height: fullScreenItem.height * (fullScreenItem.landscape ? Device.keyboardHeightLandscape
                                                                   : Device.keyboardHeightPortrait)
+	anchors.horizontalCenter: parent.horizontalCenter

         visible: true
